Update v2023.2.x

Djfe opened the pull request:
Add workarounds for mt7915 devices
improve lots of other stuff

Djfe committed:
Makefile: generate profiles.json only while

not building single devices

Djfe committed:
site.conf filter membership reports

Set variable to default value true
This should reduce batman management traffic significantly

Djfe committed:
site.mk remove BROKEN workaround

Djfe committed:
Add workarounds for mt7915 devices

ramips-mt7621: reload firmware twice daily
mediatek-filogic: reboot three times daily

Djfe committed:
Update Gluon to latest v2023.2.x

update patch numbering

maurerle commented on patches/3223.patch:
as this is merged, you can remove the patch :)

Djfe committed:
Makefile: fix issue when upstream adds new patches

no "similarity" stuff anymore in the diff that is put into .patch files.
It reorders our gluon-module patches automatically now.
Our patches follow after upstream patches by adding a letter to the
beginning of the filename. (I chose "x")
